resources/measures/HPXMLtoOpenStudio/resources/materials.rb

```diff
@@ -185,52 +185,74 @@ def self.Concrete(thick_in)
     return new(name = "Concrete #{thick_in} in.", thick_in = thick_in, mat_base = BaseMaterial.Concrete, k_in = nil, rho = nil, cp = nil, tAbs = 0.9)
   end
 
+  def self.solar_abs_map
+    return {
+      'Dark' => 0.95,
+      'MediumDark' => 0.85,
+      'Medium' => 0.70,
+      'Light' => 0.50,
+      'Reflective' => 0.30
+    }
+  end
+
   def self.ExtFinishStuccoMedDark
-    return new(name = 'Stucco, Medium/Dark', thick_in = 1.0, mat_base = BaseMaterial.Stucco, k_in = nil, rho = nil, cp = nil, tAbs = 0.9, sAbs = 0.75, vAbs = 0.75)
+    sAbs = self.solar_abs_map['MediumDark']
+    return new(name = 'Stucco, Medium/Dark', thick_in = 1.0, mat_base = BaseMaterial.Stucco, k_in = nil, rho = nil, cp = nil, tAbs = 0.9, sAbs = sAbs, vAbs = sAbs)
   end
 
   def self.ExtFinishStuccoLight
-    return new(name = 'Stucco, Light', thick_in = 1.0, mat_base = BaseMaterial.Stucco, k_in = nil, rho = nil, cp = nil, tAbs = 0.45, sAbs = 0.75, vAbs = 0.75)
+    sAbs = self.solar_abs_map['Light']
+    return new(name = 'Stucco, Light', thick_in = 1.0, mat_base = BaseMaterial.Stucco, k_in = nil, rho = nil, cp = nil, tAbs = 0.45, sAbs = sAbs, vAbs = sAbs)
```

Comment:
`tAbs = 0.45` seems wrong.

Reply:
This [comment](https://github.com/NREL/resstock/pull/561#issuecomment-861804196) implies solar absorptance should be 0.45.